ingestion/src/metadata/orm_profiler/metrics/static/min.py

```diff
@@ -30,4 +31,6 @@ def name(cls):
 
     @_label
     def fn(self):
+        if not is_quantifiable(self.col.type):
+            return None
```

Comment:
temporarily disable min/max computation for non numerical columns as otherwise backend API cannot handle the results

Reply:
@pmbrull do you see any value in taking min/max for non numerical columns?